src/pages/RFOX/components/Overview/StatItem.tsx

```diff
@@ -0,0 +1,76 @@
+import { ArrowDownIcon, ArrowUpIcon } from '@chakra-ui/icons'
+import { Skeleton, Stack, Tag } from '@chakra-ui/react'
+import { bnOrZero } from '@shapeshiftoss/chain-adapters'
+import { useMemo } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { Amount } from 'components/Amount/Amount'
+import { HelperTooltip } from 'components/HelperTooltip/HelperTooltip'
+import { Text } from 'components/Text'
+
+type StatItemProps = {
+  description: string
+  amountUserCurrency?: string
+  percentChangeDecimal?: string
+  helperTranslation?: string
+}
+
+type ChangeTagProps = {
+  value?: string
```

Comment:
Any reason to make this optional? We need a value to display a tag - if the intent is to handle loading state, then either pass a required `isLoading`, or alternatively keep it type safe and wrap in `<Skeleton />` in consumers